web/pgadmin/static/js/PgTreeView/index.jsx

```diff
@@ -38,9 +38,11 @@ const Root = styled('div')(({theme}) => ({
 
 export const PgTreeSelectionContext = React.createContext();
 
-export default function PgTreeView({ data = [], hasCheckbox = false, selectionChange = null}) {
+export default function PgTreeView({ data = [], hasCheckbox = false,
+  selectionChange = null, NodeComponent = null, ...props }) {
 
   let treeData = data;
+  NodeComponent = NodeComponent || DefaultNode;
```

Comment:
props should not be changed, may be i missed in last review.
So you can
```
const Node = NodeComponent || DefaultNode;
```
Use `??` instead of `||` please.